schemas/triniti/sys/mixin/flagset/latest.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<pbj-schema xmlns="http://gdbots.io/pbj/xsd"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://gdbots.io/pbj/xsd http://gdbots.io/pbj/xsd/schema.xsd">
+
+  <schema id="pbj:triniti:sys:mixin:flagset:1-0-0" mixin="true" extends="gdbots:ncr:mixin:node:v1">
+    <fields>
+      <field name="_id" type="identifier" required="true" overridable="true">
+        <php-options>
+          <imports>use Triniti\Schemas\Sys\FlagsetId;</imports>
+          <class-name>FlagsetId::class</class-name>
+        </php-options>
+        <js-options>
+          <imports>import FlagsetId from '@triniti/schemas/triniti/sys/FlagsetId';</imports>
+          <class-proto>FlagsetId</class-proto>
+        </js-options>
+      </field>
+      <field name="value" type="string"/>
```

Comment:
@hunyoboy you need values... a map... string_flags, bool_flags, int_flags,.  a single value is not meaningful in a flagset.  it's a map of values for a given datatype